ios/Podfile.lock

```diff
@@ -248,4 +248,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 0d3a33be0c26b65700ef25b55314f395972202ba
 
-COCOAPODS: 1.7.5
```

Comment:
This does not seem like it should be part of this PR?

Reply:
I could make a separate PR for it, but it might as well be a part of this one